database/migrations/2016_07_25_052444_AlterTableComponentGroupsAddVisibleColumn.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterTableComponentGroupsAddVisibleColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('component_groups', function (Blueprint $table) {
+            $table->integer('visible')->after('order')->unsigned()->default(0);
+
+            $table->index('visible');
+            $table->engine = 'InnoDB';
```

Comment:
Please move this to the top of the closure with a blank line under it :+1: 

(as seen here: https://github.com/CachetHQ/Cachet/blob/2.4/database/migrations/2015_01_05_203341_CreateUsersTable.php#L24)